scale-tests/run_perfdash.sh

```diff
@@ -5,9 +5,10 @@
 set -xeo pipefail
 
 export SERVER_ADDRESS="0.0.0.0:8080"
-export BUILDS="100"
+export BUILDS="10"
```

Comment:
Nit. It could make sense to change this to:

```suggestion
export BUILDS="${BUILDS:-10}"
```

To allow easily overriding the number of builds without having to modify the script. And similarly for the other parameters, when relevant (I guess mostly the server address, bucket and config path).

Reply:
makes sense, done and tested.